.azure-pipelines/azure-pipelines-external.yml

```diff
@@ -127,6 +127,14 @@ jobs:
       python -m black --check arviz examples asv_benchmarks
     displayName: 'black'
 
+  - script: |
+      python -m mypy .
```

Comment:
would it help with anything/is it possible to run mypy only on main arviz and example folders?